steps/rabbit/session.go

```diff
@@ -254,18 +254,25 @@ func matchMessage(msg string, expectedProps map[string]interface{}) bool {
 	return true
 }
 
-// WaitForMessageWithStandardProperties waits for a message with standard rabbit properties that are equal to the expected values.
-func (s *Session) WaitForMessageWithStandardProperties(ctx context.Context, timeout time.Duration, props amqp.Delivery) error {
+// WaitForMessagesWithStandardProperties waits for 'count' messages with standard rabbit properties that are equal to the expected values.
+func (s *Session) WaitForMessagesWithStandardProperties(ctx context.Context, timeout time.Duration, count int, props amqp.Delivery) error {
 	return waitUpTo(timeout, func() error {
+		n := count
```

Comment:
The variable "n" is not really needed, is it?
The rest of the things I was commenting on, cesar has included in his review.

Reply:
Why not? is the variable been decreased, isn't it?

Reply:
Understood. Done